### PR TITLE
connects AgileVentures/MetPlus_tracker#166

### PR DESCRIPTION
**Pull request by amba178**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -80,4 +80,6 @@
   config.action_mailer.delivery_method = :sendmail
   #this will take care of devise setup default url in production
   config.action_mailer.default_url_options = { :host => 'metplus.org' }
-end
```
Comment (joaopapereira) [inline config/environments/production.rb]: Did you remove the end for any particular reason?

**Comment by joaopapereira** [comment]
If in this PR we could get some users on the seed file it would help us to show the client the app in heroku, can you provide them @amba178 ?

File: config/environments/production.rb (hunk 1 above)
Comment (amba178) [inline config/environments/production.rb]: Thanks, for pointing out.

**Comment by amba178** [comment]
@joaopapereira: Sure, I can create some users.

**Comment by amba178** [comment]
Error due to ruby version, try to close and open again.

**Pull request closed without merging**